packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -125,10 +125,12 @@ class ListTileTheme extends InheritedTheme {
   /// If [ListTile.tileColor] is provided, [tileColor] is ignored.
   final Color? tileColor;
 
+  /// {@template flutter.material.ListTile.selectedTileColor}
   /// If specified, defines the background color for `ListTile` when
```

Comment:
There's non need to reiterate `ListTile` in a statement like this. Could just be:
If non-null, defines the background color when [ListTile.selected] is true.

Reply:
I've done the changes.